src/main/java/org/infoshareacademy/javaholics/Route.java

```diff
@@ -0,0 +1,113 @@
+package org.infoshareacademy.javaholics;
+
+import java.io.File;
+
+public class Route {
+
+    private long id;
+    private String name;
+    private String locality;
+    private String placeStart;
+    private String placeStop;
+    private RouteDifficulty difficulty;
+    private File routeFile;
```

Comment:
Na razie ok, pytanie czy poznej nie powinian to być Path, nie mam takiej wiedzy co bedzie lepsze